Introduce multi host in pulsar doc (#4546)
apache/pulsar: site2/docs/admin-api-overview.md

PulsarAdmin admin = new PulsarAdmin(url, config);
```

> If you have mutilple brokers to using, you can use multi host like pulsar service, like this:

Reviewer: ```suggestion
If you have multiple brokers to use, you can use multi-host like Pulsar service. For example,
```

```java
URL url = new URL("http://localhost:8080,localhost:8081,localhost:8082");

> // Pass auth-plugin class fully-qualified name if Pulsar-security enabled

Reviewer: ```suggestion
// Pass auth-plugin class fully-qualified name if Pulsar-security is enabled.
```

Reviewer: Are "auth-plugin" and "Pulsar-security" something like a prop noun?

Author: I think not.
